File: Java Programs (CompSci 12)/Java Programs (CompSci 12)/Assignment6A/src/Assignment6A.java
import java.util.Scanner;

class Assignment6A
{
	public static  void main ( String[]   args)	
	{
		Scanner input = new Scanner (System.in);

		int num1,desicion=0,num2=1;

		while (num2==1) 

		{

			System.out.println("Type in a number that you want to see if it's odd or even"); 
			num1=input.nextInt();

			desicion=num1%2;
//even
			if (desicion==0)
				System.out.println("Even");
//odd			
			if (desicion==1)
				System.out.println("Odd");
			
			System.out.println("Would you like to continue?"); 
			num2=input.nextInt();
}}}

File: School_Assignments/School_Assignments/Assignment6a.py

def add(num1,num2) :
    answer=num1+num2
    return answer
def multiply(num1,num2) :
    answer=num1*num2
    return answer
def subtract(num1,num2) :
    answer=num1-num2
    return answer
def divide(num1,num2) :
    answer=num1/num2
    return answer

num1=int(input('Type in a number'))
num2=int(input('Type in a second number'))
function=int(input('Type in your function, 1=add, 2=multiply, 3=subtract, 4=divide'))

if(function==1):
    answer=add(num1,num2)
elif(function==2):
    answer=multiply(num1,num2)
elif(function==3):
    answer=subtract(num1,num2)
elif(function==4):
    answer=divide(num1,num2)

print('The answer is', answer)
